aikau/src/test/resources/testApp/js/aikau/testing/templates/MockXhr.html

```diff
@@ -1,4 +1,4 @@
-<div class="alfresco-testing-MockXhr alfresco-testing-MockXhr--hide-body" data-dojo-attach-point="containerNode">
+<div class="alfresco-testing-MockXhr alfresco-testing-MockXhr" data-dojo-attach-point="containerNode">
    <table class="log">
```

Comment:
If we're going to change this, we need to remove `alfresco-testing-MockXhr--hide-body` entirely, as there are otherwise two identical classnames in there now.